lib/travis/build/addons/sauce_connect/templates/sauce_connect.sh

```diff
@@ -53,7 +53,10 @@ function travis_start_sauce_connect() {
   ${sc_dir}/bin/sc \
     ${sc_tunnel_id_arg} \
     -f ${sc_readyfile} \
-    -l ${sc_logfile} &
+    -l ${sc_logfile} \
+    -B ${SAUCE_NO_SSL_BUMP_DOMAINS} \
```

Comment:
I was hoping an empty argument would mean it wouldn't do anything differently. I will explore to make sure it doesn't break anything.